### DISCUSSION TITLE
Atmosphere won't load when sys-con is present on the SD card

### DISCUSSION THREAD

**Discussion by scottmeup** [cathery/sys-con#218] [Troubleshooting]
**Description**
Atmosphere won't load when sys-con is present on the SD card

**How to reproduce**
1. Extract sys-con-0.6.2 to root of the SD card
2. Boot into atmosphere
3. After atmosphere splash screens and switch logo, an error message appears along with a message to press power button to reboot: https://imgur.com/a/g1cQOXY

**Environment**
- Switch FW 10.1.0
- Atmosphere 0.14.2
- sys-con-0.6.2
- No other sysmodules enabled

**Comment by Skonikol** [comment]
> **Description**
> Atmosphere won't load when sys-con is present on the SD card
> 
> **How to reproduce**
> 
> 1. Extract sys-con-0.6.2 to root of the SD card
> 2. Boot into atmosphere
> 3. After atmosphere splash screens and switch logo, an error message appears along with a message to press power button to reboot: https://imgur.com/a/g1cQOXY
> 
> **Environment**
> 
> * Switch FW 10.1.0
> * Atmosphere 0.14.2
> * sys-con-0.6.2
> * No other sysmodules enabled

Check for initialisation files SD/config/sys-con/

**Comment by scottmeup** [comment]
Hi, this is the contents of that directory:

```
$ ls config/sys-con
config_dualshock3.ini config_xbox360.ini    example.ini
config_dualshock4.ini config_xboxone.ini
config_global.ini     config_xboxorig.ini
```

Are some changes required?
Cheers - Scott.

**Comment by Skonikol** [comment]
> Hi, this is the contents of that directory:
> 
> ```
> $ ls config/sys-con
> config_dualshock3.ini config_xbox360.ini    example.ini
> config_dualshock4.ini config_xboxone.ini
> config_global.ini     config_xboxorig.ini
> ```
> 
> Are some changes required?
> Cheers - Scott.

Is your sys-con configuration folder located in the config folder in the root directory (not atmosphere)? Check it.

**Comment by scottmeup** [comment]
Yeah, the above output is run from SD root.
The full path to those files is SD/config/sys-con, where SD is the mount point for the SD card.

**Comment by Skonikol** [comment]
I had a similar error and it was solved by fixing the location of the configuration folder. If you have everything in order with the placement, then I can only suggest using the fix of archive bit on this folder and files (especially if you are using macOS).
